src/redux/dragons/dragon.js

```diff
@@ -1,4 +1,4 @@
-const dragonReducer = (state, action) => {
+const dragonReducer = (state = 'test-dragons', action) => {
```

Comment:
This is a good way to initialize the state but maybe using initialState Variable would make the code more structured. 
For Example,
`
initialState = [];

const dragonReducer = (state = initialState, action)`